Utilities/Mergers/src/MergerBuilder.cxx

```diff
@@ -116,10 +116,7 @@ framework::DataProcessorSpec MergerBuilder::buildSpec()
 
   merger.inputs.push_back({"timer-publish", "TMR", mergerDataDescription(mName), mergerSubSpec(mLayer, mId), framework::Lifetime::Timer, timerSpecs(timers)});
   merger.labels.push_back(mergerLabel());
-  if (mConfig.expendable) {
-    framework::DataProcessorLabel expendableLabel = {"expendable"};
-    merger.labels.emplace_back(expendableLabel);
-  }
+  merger.labels.insert(merger.labels.end(), mConfig.labels.begin(), mConfig.labels.end());
```

Comment:
maybe you should also do `std::unique`?

Reply:
I did not realize it was a vector. Ok, done.